src/hardware/mixer.cpp

```diff
@@ -921,7 +833,7 @@ void MIXER_Init(Section* sec) {
 
 	mixer.nosound=section->Get_bool("nosound");
 	mixer.freq = static_cast<uint32_t>(section->Get_int("rate"));
-	mixer.blocksize = static_cast<uint16_t>(section->Get_int("blocksize"));
+	mixer.latency = static_cast<uint16_t>(section->Get_int("latency"));
```

Comment:
We've got the default range capped from 1 to 100. 
Suggest using a uint8_t to store (and cast here). No to save space.. but because it informs maintainers that we never expect a value beyond 255.. and even if the code goes off the rails - the worst case it will only ever be 255.